src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -320,8 +321,19 @@ public void init(boolean forceNewConnection) {
         clearInstances(instances);
         clearInstances(brokenInstances);
 
+
         Reporter reporter = appConfig.getReporter();
 
+        String action = appConfig.getAction();
+
+        if( action.equals( AppConfig.ACTION_LIST_JVMS )) {
+            List<VirtualMachineDescriptor> descriptors = com.sun.tools.attach.VirtualMachine.list();
+            for( VirtualMachineDescriptor descriptor : descriptors ) {
```

Comment:
What do you think about adding a top message before listing alls JVMs to emphasize  that only the ones associated to the user are listed. Something like,
```
List of JVMs for $user:
  JVM id $id: $name
  [...]
```